src/main/java/nl/novi/Les11SpringBootTechItEasyApplication/services/TelevisionService.java

```diff
@@ -0,0 +1,109 @@
+package nl.novi.Les11SpringBootTechItEasyApplication.services;
+import nl.novi.Les11SpringBootTechItEasyApplication.exceptions.RecordNotFoundException;
+import nl.novi.Les11SpringBootTechItEasyApplication.models.Television;
+import nl.novi.Les11SpringBootTechItEasyApplication.repositories.TelevisionRepository;
+import org.springframework.stereotype.Service;
+import java.util.List;
+
+@Service
+public class TelevisionService {
+    private final TelevisionRepository televisionRepository;
+
+    public TelevisionService(TelevisionRepository televisionRepository) {
+        this.televisionRepository = televisionRepository;
+    }
+
+    public List<Television> getTelevisions(){
+        return televisionRepository.findAll();
+    }
+
+    public Television getTelevisionById(Long id) {
+        if(televisionRepository.findById(id).isPresent()){
+            return televisionRepository.findById(id).get();
+        } else {
+            throw new RecordNotFoundException("The entered value isn't correct or doesn't exist. Search again with another value.");
+        }
+    }
+
+    public Television saveTelevision(Television newTv) {
+        return televisionRepository.save(newTv);
+    }
+
+    public Television updateTelevision(Television updatedTv, Long id) {
+        Television tv = getTelevisionById(id);
+
+        if(!tv.getId().equals(updatedTv.getId())) {
```

Comment:
Heel netjes dat je controleert of er een verschil is tussen de bestaande data in de database en de nieuw ontvangen data van de gebruiker voordat je data gaan wijzigen.